murmurhash3.js

```diff
@@ -0,0 +1,65 @@
+/* eslint no-fallthrough: 0, default-case: 0 */
```

Comment:
What was the reason to pick this lib over that simple function you shown me before? If we go for a lib, let's include it as npm module and allow it to be replaced through a wrapper or something. What do you think?